zerver/lib/integrations.py

```diff
@@ -327,6 +327,7 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
     WebhookIntegration('pivotal', ['project-management'], display_name='Pivotal Tracker'),
     WebhookIntegration('raygun', ['monitoring'], display_name="Raygun"),
     WebhookIntegration('reviewboard', ['version-control'], display_name="ReviewBoard"),
+    WebhookIntegration('semaphore2', ['continuous-integration', 'deployment'], stream_name='pipelines'),
```

Comment:
@punchagan Should i add docstring to `WebhookIntegration` ? (as per the 1st point)

Reply:
@punchagan could you please elaborate the first point. I mean i don't understand where exactly i need to add docs ?

Reply:
I'm also not entirely sure, but looking at the `stream_name` argument, I couldn't immediately figure out what it actually does. May be on the `Integration` class or the `WebhookIntegration` class? It can come as a separate commit.